python/ray/tune/suggest/bohb.py

```diff
@@ -312,7 +312,7 @@ def resolve_value(par: str, domain: Domain
 
         cs = ConfigSpace.ConfigurationSpace()
         for path, domain in domain_vars:
-            par = "/".join(path)
+            par = "/".join([str(p) for p in path])
```

Comment:
```suggestion
            par = "/".join(str(p) for p in path)
```
can technically be a generator comprehension I think :)